### PR TITLE
updating model cards token classification -> ner; summarization -> ne…

### PR DESCRIPTION
**Pull request by jpmann**
1. Renamed Token Classification to Named Entity Recognition and correspondingly updated the notebook and shell script names
2. Renamed Summarization use case to News Summary
3. Renamed Translation use case to Translate Romanian to English and correspondingly updated the notebook and shell script names

### CODE REVIEW

File: models/system/facebook-bart-large-cnn/description.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ Batch | coming soon
...
-Translation|Translation|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">WMT16</a>|<a href="https://aka.ms/azureml-ft-sdk-translation" target="_blank">translation.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-translation" target="_blank">translation.sh</a>
+Summarization|News Summary|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">CNN DailyMail</a>|<a href="https://aka.ms/azureml-ft-sdk-news-summary" target="_blank">news-summary.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-news-summary" target="_blank">news-summary.sh</a>
+Translation|Translate Romanian to English|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">WMT16</a>|<a href="https://aka.ms/azureml-ft-sdk-translation" target="_blank">translate-romanian-to-english.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-translation" target="_blank">translate-romanian-to-english.sh</a>
```
Comment (novaturient95) [inline models/system/facebook-bart-large-cnn/description.md:19]: should this be english to romanian? 

image

File: models/system/sshleifer-distilbart-cnn-12-6/description.md [hunk 2]
```diff
@@ -15,8 +15,8 @@ Batch | coming soon
...
-Translation|Translation|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">WMT16</a>|<a href="https://aka.ms/azureml-ft-sdk-translation" target="_blank">translation.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-translation" target="_blank">translation.sh</a>
+Summarization|News Summary|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">CNN DailyMail</a>|<a href="https://aka.ms/azureml-ft-sdk-news-summary" target="_blank">news-summary.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-news-summary" target="_blank">news-summary.sh</a>
+Translation|Translate Romanian to English|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">WMT16</a>|<a href="https://aka.ms/azureml-ft-sdk-translation" target="_blank">translate-romanian-to-english.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-translation" target="_blank">translate-romanian-to-english.sh</a>
```
Comment (novaturient95) [inline models/system/sshleifer-distilbart-cnn-12-6/description.md:19]: `Translate english to romanian` here as well?

File: models/system/t5-base/description.md [hunk 3]
```diff
@@ -14,8 +14,8 @@ Batch | coming soon
...
-Translation|Translation|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">WMT16</a>|<a href="https://aka.ms/azureml-ft-sdk-translation" target="_blank">translation.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-translation" target="_blank">translation.sh</a>
+Summarization|News Summary|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">CNN DailyMail</a>|<a href="https://aka.ms/azureml-ft-sdk-news-summary" target="_blank">news-summary.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-news-summary" target="_blank">news-summary.sh</a>
+Translation|Translate Romanian to English|<a href="https://huggingface.co/datasets/cnn_dailymail" target="_blank">WMT16</a>|<a href="https://aka.ms/azureml-ft-sdk-translation" target="_blank">translate-romanian-to-english.ipynb</a>|<a href="https://aka.ms/azureml-ft-cli-translation" target="_blank">translate-romanian-to-english.sh</a>
```
Comment (novaturient95) [inline models/system/t5-base/description.md:18]: `translation_en_ro_finetune_job` 

image

Comment (jpmann) [reply models/system/t5-base/description.md:18]: I believe this is part of azureml-examples right? Will update

File: models/system/sshleifer-distilbart-cnn-12-6/description.md (hunk 2 above)
Comment (jpmann) [reply models/system/sshleifer-distilbart-cnn-12-6/description.md]: Updated

File: models/system/facebook-bart-large-cnn/description.md (hunk 1 above)
Comment (jpmann) [reply models/system/facebook-bart-large-cnn/description.md]: Updated

**Review by novaturient95 — APPROVED** [review]

**Pull request merged**